migrations/0012_separate_bet_token_value.up.sql

```diff
@@ -0,0 +1,16 @@
+ALTER TABLE game_sessions
+  ADD COLUMN symbol VARCHAR(10) DEFAULT '4,BET',
```

Comment:
this will lock the table
https://www.postgresql.org/docs/9.4/sql-altertable.html
though I think it won't take long (depends on the data size and disk speed)

Reply:
a better way to do it is explained here
https://www.citusdata.com/blog/2018/02/22/seven-tips-for-dealing-with-postgres-locks/

Reply:
UPD: game_sessions is only at 100 MB so we can update it in whatever way we like

Reply:
thx